docs/howto/features/imagebuilding.md

```diff
@@ -0,0 +1,153 @@
+# Make an imagebuilding hub
+
+We can support users who want to build, push and launch user images, from open source GitHub repositories from within their JupyterHubs, similar with what [mybinder.org](https://mybinder.org) does, but from their own JupyterHubs.
+
+We call this an `imagebuilding` style hub and the primary features offered would be:
+
+1. User authentication
+2. Persistent storage
+3. Server options allowing:
+  - pre-defined list of images to choose from
+  - specifying another image, different from the ones in the pre-defined list
+  - building and pushing an image to a registry from a GitHub repository
+
+## Connect with `jupyterhub-fancy-profiles`
+
+1. The [jupyterhub-fancy-profiles](https://github.com/yuvipanda/jupyterhub-fancy-profiles) project provides a user facing frontend for connecting the JupyterHub to the BinderHub service, allowing the users to build their own images similar with how they would on mybinder.org.
+
+```yaml
+hub:
+  extraConfig:
```

Comment:
Just to note that if we still need to keep a separate image for this, that should be mentioned as well.

I'm also not 100% sure, but this code block may need to be indented 3 spaces to line up with the numbered list, so it is counted as *part* of the item rather than just a separate code block?

Reply:
The code block formatting suggestion applies to all code blocks here. But if it renders visually ok, I don't particularly care otherwise :)